pycheribuild/config/compilation_targets.py

```diff
@@ -742,6 +788,13 @@ class CompilationTargets(BasicCompilationTargets):
     CHERIBSD_RISCV_PURECAP = CrossCompileTarget("riscv64-purecap", CPUArchitecture.RISCV64, CheriBSDTargetInfo,
                                                 is_cheri_purecap=True, hybrid_target=CHERIBSD_RISCV_HYBRID)
     CHERIBSD_AARCH64 = CrossCompileTarget("aarch64", CPUArchitecture.AARCH64, CheriBSDTargetInfo)
+    # XXX: Do we want a morello-nocheri variant that uses the morello compiler for AArch64 instead of CHERI LLVM?
```

Comment:
A morello-nocheri target might be useful for performance comparisons, however I don't expect we will run comparisons on the simulator.

Reply:
I guess it might also be useful for Jenkins purposes so that we can build aarch64 world without pulling in CHERI LLVM.